mu-plugins/central-hub/tests/phpunit/unit/config-store/_loadConfigFromFilesystem.php

```diff
@@ -45,4 +46,36 @@ public function test_should_return_array_when_path_to_file_is_given() {
 		$this->assertSame( 'foo', $actual[0] );
 		$this->assertSame( $expected, $actual[1] );
 	}
+
+	/**
+	 * Test _load_config_from_filesystem() should throw an Exception when configuration store key is empty
+	 *  and parameters exist.
+	 */
+	public function test_should_throw_exception_when_no_store_key() {
+		$this->expectException( \Exception::class );
+		$this->expectExceptionMessage(
+			sprintf( 'No store key exists in the [path_to_file] configuration file.', 'path_to_file' )
```

Comment:
@rgadon107 What's wrong here?  

You need to insert the path to file inside of the message's string.  You use `sprintf` to replace a placeholder with the value from a variable:

```php
sprint( 'some message %s', $some_variable )
```

So the placeholder is `%s` and it is replaced by the value associated with the variable.

Now, let's relate that concept to this edge case.

The message the [error will throw is found here](https://github.com/rgadon107/cornerstone/pull/20/files#diff-aff0ac70304193f0d42b37b100807912R77):

```php
function _load_config_from_filesystem( $path_to_file ) {
	$config = (array) require $path_to_file;
	$config    = (array) require $path_to_file;
	$store_key = key( $config );
	if ( empty( $store_key ) ) {
		throw new \Exception(
			sprintf( 'No store key exists in the %s configuration file.', esc_attr( $path_to_file ) )
		);
	}
```

Notice the structure of the message:

1. It needs a variable to replace out the placeholder of `%s`.
2. There's no square brackets around the path to file.

Put it all together and the message that will be thrown is:

```
No store key exists in the path/to/file configuration file.
```

Now you need to replace out that `path/to/file` with the right path to file in the test.  How do you do that?

1. Notice that `$path_to_file` is below this exception.  You need that variable.  Reorder by moving that line of code up to the top.
2. Fix the `sprintf`.

I fixed this in commit https://github.com/rgadon107/cornerstone/pull/20/commits/e1e68716f9375ac62efe610c8599180f4686b255